src/main/scala/com/samstarling/prometheusfinagle/PrometheusStatsReceiver.scala

```diff
@@ -7,12 +7,16 @@ import scala.collection.concurrent.TrieMap
 import com.twitter.finagle.util.DefaultTimer
 import com.twitter.util._
 
+private object CustomDefaultTimer {
+  def getInstance: Timer = Option(DefaultTimer.getInstance).getOrElse(new JavaTimer())
```

Comment:
Might want to make this `daemon=true`? That's [what Finagle does](https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/util/DefaultTimer.scala#L84).